control-plane-operator/controllers/hostedcontrolplane/olm/assets/packageserver-endpoint-guest.yaml

```diff
@@ -4,8 +4,7 @@ metadata:
   name: packageserver
   namespace: default
 subsets:
-- addresses:
-  - ip: {{ .PackageServerAPIClusterIP }}
```

Comment:
Could you explain what this change does? Thanks!

Reply:
The manifest is no longer a go template. The package server cluster IP is now set here:
https://github.com/openshift/hypershift/pull/518/files#diff-b1df563b312192d05f3b030fc9c3beac30c393525674a12c00450e77d652008bR42-R47